keyboards/handwired/rabijl/rotary_numpad/keymaps/default/config.h

```diff
@@ -0,0 +1,5 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+
+#define ENCODER_MAP_ENABLE
```

Comment:
This should be in the keymap's rules.mk:
```make
ENCODER_MAP_ENABLE = yes
```

And this file can be removed.

Reply:
so in this case a makefile is allowed in the keymap dir? I thought that wasn't allowed anymore according to this: https://docs.qmk.fm/#/contributing?id=keymaps

Reply:
To clarify, once upon a time, the keymap folders had `makefile` in them, rather than `rules.mk`.  And occasionally we see people using `makefile` instead of `rules.mk`.   Since the build process doesn't support `makefile` there anymore, we try to make sure that people don't include them. 

However, rules.mk files are absolutely allowed in keymap folders.  Though, generally, we discourage heavy usage of them for the default keymap, this is one of those cases that it's necessary.

Reply:
Alright I've resolved the issue